packages/cli/src/commands/push.ts

```diff
@@ -141,23 +144,37 @@ async function collectFilesToUpload(entrypoint: string) {
 
   if (config.configFile) {
     files.push(getFileEntry(config.configFile));
+    if (fs.existsSync('package.json')) { files.push(getFileEntry('package.json')); }
+    if (fs.existsSync(IGNORE_FILE)) { files.push(getFileEntry(IGNORE_FILE)); }
     if (config.referenceDocs.htmlTemplate) {
       const dir = getFolder(config.referenceDocs.htmlTemplate);
       const fileList = getFilesList(dir, []);
       files.push(...fileList.map(f => getFileEntry(f)));
     }
     if (config.rawConfig && config.rawConfig.lint && config.rawConfig.lint.plugins) {
+      let pluginFiles = new Set();
```

Comment:
```suggestion
      let pluginFiles = new Set<string>();
```

You wont need `as string[]` then below.